### PR TITLE
Disable generate ssh on low balance

### PR DESCRIPTION
**Pull request by zaelgohary**
### Description

Disable generate ssh if balance is less than 0.01

### Related Issues

- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/322

### Checklist

- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [ ] Code format and docstrings
- [ ] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

File: packages/playground/src/weblets/profile_manager.vue [hunk 1]
```diff
@@ -221,7 +221,7 @@
           <VBtn
             color="secondary"
             variant="text"
-            :disabled="!!ssh || updatingSSH || generatingSSH"
+            :disabled="!!ssh || updatingSSH || generatingSSH || (balance && balance.free < 0.01)"
```
Comment (Mahmoud-Emad) [inline packages/playground/src/weblets/profile_manager.vue:224]: I believe that creating a helper function to check the balance and determine if it is sufficient or not would be beneficial in terms of reusability.

```ts

const isEnoughBalance = (balance: number) => {
    // Check the free balance and return true if it's enough for making transactions or not
    return (balance && balance.free < 0.01) ? true : false
}
```
```suggestion

            :disabled="!!ssh || updatingSSH || generatingSSH || !isEnoughBalance()"
```

Comment (AhmedHanafy725) [inline packages/playground/src/weblets/profile_manager.vue:224]: is it 0.01 or 0.001?

Comment (zaelgohary) [reply packages/playground/src/weblets/profile_manager.vue:224]: Edited.

**Review by AhmedHanafy725 — APPROVED** [review]

**Review by Mahmoud-Emad — APPROVED** [review]
LGTM, Good job ya zizi :)

**Pull request merged**